### PR TITLE
[Ready]Deploy appveyor builds

### PR DESCRIPTION
**Pull request by ShFil119**
Not sure if we also want to deploy builds from pulls.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -38,3 +38,25 @@ init:
 
 build_script:
   - ctest -VV -S cmake/ctest/script_ci.ctest
+
+# scripts to run after build
+after_build:
+  - 7z a openrw-%appveyor_repo_branch%.zip %APPVEYOR_BUILD_FOLDER%\build\rwgame\Release\
```
Comment (husho) [inline .appveyor.yml]: Consider adding ```*``` in the end of the path to get rid of unnecessary folders in archive

**Comment by ShFil119** [comment]
@madebr suggested that it's good step to move setting CMAKE_INSTALL_PREFIX into ctest. 
I'm trying to do this. https://github.com/ShFil119/openrw/pull/35/files

Which version do you prefer?

**Comment by husho** [comment]
I'm not familiar with cmake, so I can't speak for myself.

File: .appveyor.yml [hunk 2]
```diff
@@ -38,11 +38,11 @@ init:
 
 build_script:
   - ctest -VV -S cmake/ctest/script_ci.ctest
+  - msbuild %APPVEYOR_BUILD_FOLDER%\build\rwgame\INSTALL.vcxproj
```
Comment (madebr) [inline .appveyor.yml]: I believe calling `msbuild` explicitly can be changed to 
```
cmake --target install --config Release "%APPVEYOR_BUILD_FOLDER%\build"
```
(or something similar).
That would make it build system agnostic.

Comment (ShFil119) [reply .appveyor.yml]: We want to install all or "only" rwgame?

**Comment by darkf** [comment]
Is this ready?

**Comment by madebr** [comment]
Not pr ready, but I've tried creating archives using CPack.

https://github.com/madebr/openrw/commit/02866b9b91f17c91c75de4ab2de2813ae936034c

`cpack -G TXZ`

**Comment by ShFil119** [comment]
@madebr, do you want to include using cpack in this pull?

**Comment by madebr** [comment]
CPack does not need any CMAKE_INSTALL_PREFIX fu.
It can also help create installers. (Qt IFW)

**Comment by ShFil119** [comment]
Succeeded by #503.

**Pull request closed without merging**